policy/modules/admin/su.if

```diff
@@ -55,7 +55,7 @@ template(`su_restricted_domain_template', `
 	# By default, revert to the calling domain when a shell is executed.
 	corecmd_shell_domtrans($1_su_t,$2)
 	allow $2 $1_su_t:fd use;
-	allow $2 $1_su_t:fifo_file rw_file_perms;
+	allow $2 $1_su_t:fifo_file rw_fifo_file_perms;
```

Comment:
There are several domtrans interfaces that should be fixed to use `rw_inherited_fifo_file_perms`, to be consistent with what is in `domtrans_pattern()`

Reply:
I got a few of those in the first round.  I think I got them all with the latest update.